BitmarkRegistry/Services/ChibitronicsService.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ChibitronicsService.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 8/1/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import BitmarkSDK
+
+enum VerificationLinkSource {
+  case qrCode, deepLink
+}
+
+struct ChibitronicsService {
```

Comment:
I'd suggest to use `class` instead of `struct` if the object is not carrying data model.
Class works as reference type, can be inherited if needed, and we can control `deinit`.

Reply:
And the design tent to be general, it's not serving for Chibitronic solely, I'd suggest you discuss with Hieu for the naming. We should not specific this module is for Chibitronic only.

Reply:
> I'd suggest to use `class` instead of `struct` if the object is not carrying data model.
> Class works as reference type, can be inherited if needed, and we can control `deinit`.

I don't really understand which better to use between class vs struct. But as https://developer.apple.com/documentation/swift/choosing_between_structures_and_classes, I see they recommend that use struct by default and 
- Use classes when you need Objective-C interoperability.
- Use classes when you need to control the identity of the data you're modeling.

Reply:
For modeling, I agree that we should use `struct` by default.
But this is a `service`, sounds not to be a model, if we pass this service function by function, I'd expect it's passing by reference type, not value type.
Moreover, as it's a service, we need to control `deinit` event to stop on-going actions (like url request) later on if needed.

Reply:
got it, thanks so much for your information 👍